From 00277d60e98409e493f6208f7f541f7b85ace91f Mon Sep 17 00:00:00 2001
From: Zdenek Kabelac <zkabelac@redhat.com>
Date: Thu, 11 Jan 2018 16:15:16 +0100
Subject: [PATCH 16/25] pvmove: drop misleading pvmove restriction for cluster

pvmove handles properly locked LVs in cluster and this extra check
actually cause misbehavior as some LVs were silently skipped from
operation scope.

(cherry picked from commit 02621cffb0d5ff72adb8770e26c1b4adb95bd0d9)
---
 tools/pvmove.c | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/tools/pvmove.c b/tools/pvmove.c
index 9bbe12b..c5d7e52 100644
--- a/tools/pvmove.c
+++ b/tools/pvmove.c
@@ -371,16 +371,6 @@ static struct logical_volume *_set_up_pvmove_lv(struct cmd_context *cmd,
 		if (lv_name && strcmp(lv->name, top_level_lv_name(vg, lv_name)))
 			continue;
 
-		/*
-		 * RAID, thin and snapshot-related LVs are not
-		 * processed in a cluster, so we don't have to
-		 * worry about avoiding certain PVs in that context.
-		 *
-		 * Allow clustered mirror, but not raid mirror.
-		 */
-		if (vg_is_clustered(vg) && !lv_is_mirror_type(lv))
-			continue;
-
 		if (!lv_is_on_pvs(lv, source_pvl))
 			continue;
 
-- 
1.8.3.1

